service/src/main/java/de/starwit/service/impl/TrafficIncidentService.java

```diff
@@ -76,6 +80,31 @@ public List<TrafficIncidentEntity> findAllWithoutOtherTrafficIncidentType(Long i
         return trafficincidentRepository.findAllWithoutOtherTrafficIncidentType(id);
     }
 
+    public byte[] getFileFromMinio(String bucketName, String objectName) {
+        try {
+            MinioClient minioClient = MinioClient.builder()
+                    .endpoint("http://localhost:9000")
```

Comment:
please configure the parameter (minio.endpoint, minio.username, minio.password) via application.properties